src/axolotl/train.py

```diff
@@ -181,7 +181,12 @@ def terminate_handler(_, __, model):
         model.save_pretrained(cfg.output_dir, safe_serialization=safe_serialization)
 
     if not cfg.hub_model_id:
-        trainer.create_model_card(model_name=cfg.output_dir.lstrip("./"))
+        trainer.create_model_card(
```

Comment:
Very cool! Given you're adding datasets, here you can potentially also add `finetuned_from`. This is equivalent to `base_model` in the configuration from `axolotl` and will link the fine-tuned models to their base models

See https://huggingface.co/docs/hub/model-cards#specifying-a-base-model

Reply:
Actually this might already be specified :fire:

Reply:
@osanseviero should we use something like `hf_api.list_models(filter=...)` to verify that the base_model specified isn't a locally downloaded model? or is there a simpler lightweight check?

Reply:
You can go with https://github.com/huggingface/huggingface_hub/issues/36#issuecomment-838653185 (so catching an error upon doing `model_info`) - that would likely be the most precise